components/sst_importer/src/sst_writer.rs

```diff
@@ -366,18 +446,62 @@ mod tests {
 
     #[test]
     fn test_raw_write_ttl_not_enabled() {
-        let mut meta = SstMeta::default();
-        meta.set_uuid(Uuid::new_v4().as_bytes().to_vec());
+        let (mut w, _handle) = new_writer(SSTImporter::new_raw_writer, ApiVersion::V1);
```

Comment:
Nit: the following 4 test cases are all for checking api version & key & value, and are similar. Maybe they can be put together. Please refer to [here](https://github.com/tikv/tikv/blob/5d5f1656cd1f03c85cf6e59fc6feed6f89df3fac/tests/integrations/storage/test_storage.rs#L979).

Reply:
I prefer to keep the tests KISS enough and have fewer abstractions. 😆.